.gitignore

```diff
@@ -29,5 +29,7 @@ bez
 .idea
 build/
 dist/
+.eggs/
 *.egg-info/
-/fontcrunch/quadopt_wrap.cpp
+src/fontcrunch/_version.py
+src/fontcrunch/_quadopt.cpp
```

Comment:
is this a generated file? Can we comment gitignore?

Reply:
yes, it's generated by cython from the _quadopt.pyx source. I'll add a comment